src/sdl2_compat.c

```diff
@@ -1919,7 +1919,6 @@ EventFilter3to2(void *userdata, SDL_Event *event3)
         case SDL_EVENT_WINDOW_FOCUS_GAINED:
         case SDL_EVENT_WINDOW_FOCUS_LOST:
         case SDL_EVENT_WINDOW_CLOSE_REQUESTED:
-        case SDL_EVENT_WINDOW_TAKE_FOCUS:
```

Comment:
This changes the order of window events. You'll probably need a function to convert between SDL3 window events and `event2.window.event`, instead of the hack below.

Reply:
Ah, I see the SHOWN + 1 used for the SDL2 event mapping below.